AD Date Addition

```diff
@@ -0,0 +1,25 @@
+#Declare variables for Title, First Name, and Last Name
+
+$title = "Mr."
+$firstname = "John"
+$lastname = "Smith"
+
+#Connect to Active Directory
+
+Import-Module ActiveDirectory
+
+#Loop through adding multiple employees with AAA in their names
+
+for ($i=1; $i -le 5; $i++) 
+{New-ADUser -Name "$title $firstName AAA$i $lastName"
+-$GivenName, "$firstName AAA$i", -$Surname, "$lastName"
+-$SamAccountName, "$firstNameAAA$i$lastName" 
+UserPrincipalName "$firstNameAAA$i$lastName@yourdomain.com", -Path 
+"OU=Employees, DC=yourdomain, DC=com" -$AccountPassword (ConvertTo-SecureString
+"P@ssw0rd" -AsPlainText -Force) -Enabled $true }
```

Comment:
`{New-ADUser -Name "$firstName $lastName" -GivenName "$firstName" -Surname "$lastName" -SamAccountName "$firstName $lastName" -UserPrincipalName "$firstName $lastName'@yourdomain.com'" -Path "OU=Employees, DC=yourdomain, DC=com" -Enabled $true}`

I think it was just syntax, and an error with the plaintext string to secure string causing an issue. Further testing needs to be done within the environment, but I think this is good.